tee-worker/local-setup/launch.py

```diff
@@ -61,14 +61,15 @@ def main(processes, config_path, parachain_type):
         print('Starting litentry-parachain done')
         print('----------------------------------------')
 
+    UNTRUSTED_HTTP_PORT=4545
     c = pycurl.Curl()
-    c.setopt(pycurl.URL, 'http://localhost:4545/is_initialized')
-
     worker_i = 0
     worker_num = len(config["workers"])
     for w_conf in config["workers"]:
         processes.append(run_worker(w_conf, worker_i))
         print()
+        url = 'http://localhost:' + str(UNTRUSTED_HTTP_PORT+worker_i) + '/is_initialized'
```

Comment:
Actually the port should come from parsing the worker.config (I could use something other than 4545 due to e.g. port conflict), but let's leave it - it's for local use anyway

Reply:
Updated.